pkg/mons/restore_quorum.go

```diff
@@ -215,6 +217,20 @@ func updateMonMap(ctx context.Context, clientsets *k8sutil.Clientsets, clusterNa
 	exec.RunCommandInLabeledPod(ctx, clientsets, labelSelector, "mon", "monmaptool", []string{"--print", monmapPath}, clusterNamespace, false, true)
 }
 
+func fetchGoodMon(ctx context.Context, clientsets *k8sutil.Clientsets, operatorNamespace, clusterNamespace string) string {
+	monStatOutput := exec.RunCommandInOperatorPod(ctx, clientsets, "ceph", []string{"mon", "stat", "--format", "json-pretty"}, operatorNamespace, clusterNamespace, true, false)
+	type monStat struct {
+		Leader string `json:"leader"`
+	}
+
+	var monStatOut monStat
+	err := json.Unmarshal([]byte(monStatOutput), &monStatOut)
+	if err != nil {
+		logging.Fatal(fmt.Errorf("failed to unmarshal mon stat output: %v", err))
+	}
+	return monStatOut.Leader
```

Comment:
Can this be empty?

Reply:
I have not seen this value empty, but it's worth adding a check to see if it is empty or not. Thanks